packages/react-reconciler/src/ReactFiberWorkLoop.new.js

```diff
@@ -2993,64 +2999,74 @@ function flushRenderPhaseStrictModeWarningsInDEV() {
   }
 }
 
-function commitDoubleInvokeEffectsInDEV(
-  fiber: Fiber,
+function recursivelyTraverseAndDoubleInvokeEffectsInDEV(
+  root: FiberRoot,
+  parentFiber: Fiber,
+  isInStrictMode: boolean,
   hasPassiveEffects: boolean,
```

Comment:
Nit: I would remove this `hasPassiveEffects` argument entirely. It's only an optimization but the way we should do this eventually is to check if `subtreeFlags` includes `PassiveStatic`. We can save that for a follow up though because we currently don't rely on those "static" flags anywhere and it's a mildly risky change that I want to roll out gradually with a GK. In the meantime, since this is dev-only anyway, might as well simplify the code.